packages/status-js/src/client/client.ts

```diff
@@ -56,34 +60,30 @@ class Client {
 
   static async start(options: ClientOptions) {
     // Waku
-    const fleet =
-      options.environment === 'test'
-        ? discovery.predefined.Fleet.Test
-        : discovery.predefined.Fleet.Prod
+    const fleet = options.environment === 'test' ? Fleet.Test : Fleet.Prod
     /**
      * >only connects to 1 remote node because of the limited number of nodes
      * >run by Status and the limited number of connections provided by these nodes
      * >
      * >@see https://forum.vac.dev/t/waku-v2-scalability-studies/142/2
      */
     const peers = getPredefinedBootstrapNodes(fleet)
-    const waku = await Waku.create({
-      bootstrap: {
-        default: false,
-        peers,
-      },
+    const waku = await createWaku({
+      defaultBootstrap: false,
+      emitSelf: true,
```

Comment:
fixme: extend `js-waku` type

Reply:
Ah true, well spotted. Feel free to open a PR.

FYI, I have this PR in the works: https://github.com/status-im/js-waku/pull/931

Reply:
Ref https://github.com/status-im/js-waku/pull/938.